pkg/registry/registry_antrea.csv

```diff
@@ -56,3 +56,4 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 154,egressIP,string,,current,,,,,,,,56506,
 155,appProtocolName,string,,current,,,,,,,,56506,
 156,httpVals,string,,current,,,,,,,,56506,
+157,egressNode,string,,current,,,,,,,,56506,
```

Comment:
Given naming for other existing IEs (`sourceNodeName` and `destinationNodeName`), this should probably be called `egressNodeName`